podium-gateway/src/main/resources/config/application.yml

```diff
@@ -61,6 +61,12 @@ spring:
             enabled: false
     thymeleaf:
         mode: XHTML
+    security:
```

Comment:
Should be added to the [bbmri-podium-config](https://github.com/thehyve/bbmri-podium-config) as well!